coops-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -114,11 +114,11 @@ describe('Dashboard - UI', () => {
     // emit filing list from Filing History component with the completed filing
     wrapper.find(FilingHistoryList).vm.$emit('filings-list', [
       {
-        'name': 'Director Change',
```

Comment:
This file shouldn't be in your PR.
Please remove , and remember to not change others spacings.

Reply:
weird...I didn't touch this